src/cmd/build.js

```diff
@@ -60,8 +61,12 @@ function defaultPackageCreator(
           filter: (...args) => fileFilter.wantFile(...args),
         })
         .then((buffer) => {
+          let messageData: any =  readFileSync(
+             path.join(sourceDir, '_locales', manifestData.default_locale,
```

Comment:
In translation, the Flow error is saying "`manifestData.default_locale` could be undefined. This type is incompatible with `path.join(...parts: Array<string>)` (specifically, it's incompatible with the `string` part).

Does that help you pinpoint the problem you need to fix?